src/components/ModalContents/EndOfGameModalContent.js

```diff
@@ -60,7 +62,7 @@ const EndOfGameModalContent = (gameType, answersListPlayer, answersListComputer)
 
   const handleSaveScore = (name, score) => {
     if (name.length === 0) {
-      alert('Nie możesz ciągle działać pod przykrywką. Podaj swój pseudonim!');
+      alert("You can't go undercover all the time. Give us your name!");
```

Comment:
Tu myślę, że lepiej mogłoby brzmieć `alert("You can't go undercover all the time. Tell us your name!");`

Reply:
Tutaj też chciałem, żeby było tak "slangowo". Coś w stylu "dawaj imię", zamiast "powiedz nam, jak masz na imię" (tak jakby ktoś tego żądał). Jeśli tylko jest poprawnie po angielsku, to tak bym zostawił. @RadekDulisz accept?